homeassistant/components/evohome/__init__.py

```diff
@@ -461,18 +462,20 @@ async def save_auth_tokens(self) -> None:
 
         if self.client_v1:
             app_storage[USER_DATA] = {  # type: ignore[assignment]
-                "sessionId": self.client_v1.broker.session_id,
+                SZ_SESSION_ID: self.client_v1.broker.session_id,
             }  # this is the schema for STORAGE_VER == 1
         else:
             app_storage[USER_DATA] = {}  # type: ignore[assignment]
 
         await self._store.async_save(app_storage)
 
-    async def call_client_api(self, api_function, update_state=True) -> Any:
+    async def call_client_api(
+        self, api_function: Coroutine, update_state: bool = True
```

Comment:
`Coroutine` is generic. Should we add type annotations for the return value?

Reply:
Done.